### PR TITLE
Minor BabbleSim test improvements

### PR DESCRIPTION
**Pull request by asbjornsabo**
- Add text "PASSED" to output of passed tests
- Add missing newline
- Remove superfluous #includes from media tests

### CODE REVIEW

**Review by Thalley — DISMISSED** [review]

File: tests/bluetooth/bsim_bt/bsim_test_audio/src/common.h [hunk 1]
```diff
@@ -49,10 +49,10 @@
 		bs_trace_error_time_line(__VA_ARGS__); \
 	} while (0)
 
-#define PASS(...) \
+#define PASS(__str) \
 	do { \
 		bst_result = Passed; \
-		bs_trace_info_time(1, __VA_ARGS__); \
+		bs_trace_info_time(1, "PASSED: %s", __str); \
```
Comment (Thalley) [inline tests/bluetooth/bsim_bt/bsim_test_audio/src/common.h:55]: We'll get a few "PASSED: <X> passed" strings now though, which is fine but does seem overkill with "passed" twice ;)

Comment (asbjornsabo) [reply tests/bluetooth/bsim_bt/bsim_test_audio/src/common.h:55]: I can live with that.  It is better than getting none.

**Review by MariuszSkamra — CHANGES_REQUESTED** [review]

File: tests/bluetooth/bsim_bt/bsim_test_audio/src/common.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (MariuszSkamra) [reply tests/bluetooth/bsim_bt/bsim_test_audio/src/common.h:55]: Oh, you can do just
`bs_trace_info_time(1, "PASSED: " __VA_ARGS__);`

[the remark this answers is not in the document]
Comment (asbjornsabo) [reply tests/bluetooth/bsim_bt/bsim_test_audio/src/common.h]: Changed

**Review by Thalley — APPROVED** [review]

**Review by MariuszSkamra — APPROVED** [review]
LGTM

**Pull request merged**